content/cucumber/environment-variables.md

```diff
@@ -7,7 +7,7 @@ title: Environment Variables
 
 > TODO: Make more general
 
-When [Running Features](/configuration/#running-features), it can sometimes be handy to pass special
+When [Running Features](/running), it can sometimes be handy to pass special
```

Comment:
Change "Running Features" to "running features" (lowercase for consistency) or "running Cucumber" (like menu)
"Step Definitions" -> "step definitions" (same)

Reply:
(Probably all of the Ruby docs should be checked for consistency)

Reply:
Done